.eslintrc.js

```diff
@@ -29,8 +29,7 @@ module.exports = {
             { "code": 95 }
         ],
         "linebreak-style": [
-            "error",
-            "unix"
+            0
         ],
```

Comment:
(Accidentally deleted this comment earlier.)

Why are you changing the required linebreak style in .eslintrc?

Reply:
I intended to make those changes on my local repo so the linter didn't go crazy. I intended to not add it to the staging area and commit it, I must have added it by accident somehow, I'll remove it.

Reply:
Probably a better idea to configure your editor for unix-style line endings, if your editor has that feature and supports per-project configuration.